x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -101,23 +106,8 @@ export class TaskRunnerFactory {
 
         const path = addSpaceIdToPath('/', spaceId);
 
-        // Since we're using API keys and accessing elasticsearch can only be done
-        // via a request, we're faking one with the proper authorization headers.
-        const fakeRequest = KibanaRequest.from({
-          headers: requestHeaders,
-          path: '/',
-          route: { settings: {} },
-          url: {
-            href: '/',
-          },
-          raw: {
-            req: {
-              url: '/',
-            },
-          },
-        } as unknown as Request);
-
-        basePathService.set(fakeRequest, path);
+        const request = getRequest(apiKey);
```

Comment:
Can we remove lines 102-105 since that logic has moved in the to `getRequest` function?